### PR TITLE
Update .NET migration articles

### PR DESCRIPTION
**Pull request by adegeo**
TODO:

- [x] Check that WPF doesn't need more info

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [dotnet-desktop-guide/net/winforms/migration/index.md](https://github.com/dotnet/docs-desktop/blob/0289b49004bca5eeb48acb56ccb0de6b7fc4e3d0/dotnet-desktop-guide/net/winforms/migration/index.md) | [How to migrate a Windows Forms desktop app to .NET 5](https://review.learn.microsoft.com/en-us/dotnet/desktop/winforms/migration/index?branch=pr-en-us-1641&view=netdesktop-7.0) |
| [dotnet-desktop-guide/net/wpf/migration/differences-from-net-framework.md](https://github.com/dotnet/docs-desktop/blob/0289b49004bca5eeb48acb56ccb0de6b7fc4e3d0/dotnet-desktop-guide/net/wpf/migration/differences-from-net-framework.md) | [Differences in WPF .NET](https://review.learn.microsoft.com/en-us/dotnet/desktop/wpf/migration/differences-from-net-framework?branch=pr-en-us-1641&view=netdesktop-7.0) |
| [dotnet-desktop-guide/net/wpf/migration/index.md](https://github.com/dotnet/docs-desktop/blob/0289b49004bca5eeb48acb56ccb0de6b7fc4e3d0/dotnet-desktop-guide/net/wpf/migration/index.md) | [dotnet-desktop-guide/net/wpf/migration/index](https://review.learn.microsoft.com/en-us/dotnet/desktop/wpf/migration/index?branch=pr-en-us-1641&view=netdesktop-7.0) |
| [dotnet-desktop-guide/net/wpf/overview/index.md](https://github.com/dotnet/docs-desktop/blob/0289b49004bca5eeb48acb56ccb0de6b7fc4e3d0/dotnet-desktop-guide/net/wpf/overview/index.md) | [What is Windows Presentation Foundation](https://review.learn.microsoft.com/en-us/dotnet/desktop/wpf/overview/index?branch=pr-en-us-1641&view=netdesktop-7.0) |

### CODE REVIEW

File: dotnet-desktop-guide/net/wpf/migration/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+After the upgrade is complete, the results are shown. If an item has a warning symbol, it means that there's a note for you to read, which you can do by expanding the item.
+
+## Generate a clean build
+
+After your project is upgraded, clean and compile it.
+
+01. Right-click on the **WebSiteRatings** project in the **Solution Explorer** window and select **Clean**.
+01. Right-click on the **WebSiteRatings** project in the **Solution Explorer** window and select **Build**.
+
+Any errors and incompatibilities are listed in the **Error List** window.
```
Comment (OliaG) [inline dotnet-desktop-guide/net/wpf/migration/index.md:82]:
```suggestion
If your application encountered any errors, you can find them in the **Error List** window with a recommendation how to fix them.
```

File: dotnet-desktop-guide/net/wpf/overview/index.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ There are two implementations of WPF:
 
 This overview is intended for newcomers and covers the key capabilities and concepts of WPF. To learn how to create a WPF app, see [Tutorial: Create a new WPF app](../get-started/create-app-visual-studio.md).
 
-## Why migrate from .NET Framework
+## Why upgrade from .NET Framework
 
-WPF for .NET provides new features and enhancements over .NET Framework. To learn how to migrate an app, see [How to migrate a WPF desktop app to .NET 5](../migration/convert-project-from-net-framework.md).
+WPF for .NET provides new features and enhancements over .NET Framework. For more information, see [How to upgrade a WPF desktop app to .NET 7](../migration/index.md).
```
Comment (OliaG) [inline dotnet-desktop-guide/net/wpf/overview/index.md:36]:
```suggestion
When you are upgrading your application from .NET Framework to .NET, you will benefit from much better performance, new .NET APIs, the latest C# language, improved accessibility and reliability, lots of .NET platform enhancements, new updated tooling and many more. To learn how to upgrade your application, see [How to upgrade a WPF desktop app to .NET 7](../migration/index.md).
```

**Review by OliaG — COMMENTED** [review]
Looks great!

**Review by OliaG — COMMENTED** [review]
Looks great!

File: dotnet-desktop-guide/net/winforms/migration/index.md [hunk 3]
```diff
@@ -1,215 +1,94 @@
...
-      <AutoGen>True</AutoGen>
-      <DependentUpon>Resources.resx</DependentUpon>
-      <DesignTime>True</DesignTime>
-    </Compile>
-  </ItemGroup>
-  ```
-
-  > [!IMPORTANT]
-  > **Visual Basic** projects typically use the folder _My Project_ while C# projects typically use the folder _Properties_ for the default project resource file.
+Any errors and incompatibilities are listed in the **Error List** window.
```
Comment (adegeo) [inline dotnet-desktop-guide/net/winforms/migration/index.md:89]:
```suggestion
If your application encountered any errors, you can find them in the **Error List** window with a recommendation how to fix them.
```

File: dotnet-desktop-guide/net/wpf/overview/index.md (hunk 2 above)
Comment (adegeo) [reply dotnet-desktop-guide/net/wpf/overview/index.md:36]:
```suggestion
When you are upgrading your application from .NET Framework to .NET, you will benefit from:

- Better performance
- New .NET APIs
- The latest language improvements
- Improved accessibility and reliability
- Updated tooling and more

To learn how to upgrade your application, see [How to upgrade a WPF desktop app to .NET 7](../migration/index.md).
```

**Review by OliaG — APPROVED** [review]
Looks great

**Pull request merged**